R/geom-dotplot.r

```diff
@@ -208,25 +208,29 @@ GeomDotplot <- ggproto("GeomDotplot", Geom,
       stackdots <- function(a)  a - 1 - floor(max(a - 1) / 2)
       stackaxismin <- -.5
       stackaxismax <- .5
+    } else {
+      abort(glue('{params$stackdir} is not a recognized value for `stackdir`. Use "up", "down", "center", or "centerwhole"'))
```

Comment:
Use `arg_match0()`?

Reply:
has that been present in slang for a long time?